base/allocator/partition_allocator/BUILD.gn

```diff
@@ -360,6 +360,11 @@ component("partition_alloc") {
   if (enable_pkeys && is_debug) {
     configs += [ ":no_stack_protector" ]
   }
+
+  # Need to include |component("partition_alloc")| for gn check.
```

Comment:
Prefer using one of ` check_includes = false`/`friend`/`nogncheck`

Reply:
Could you add a bug to the taskflow backlog to clean this up and add that to the TODO please?